spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -35,22 +35,22 @@ It can also be used to implement a drop-in configuration approach.
 Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
 If the jar is present on the classpath then it will override configuration values from <<ConfigSource,ConfigSources>> with lower `ordinal` values.
 
-The ordinal for property file based <<ConfigSource,ConfigSources>> can be configured using the key `config_ordinal` inside the property file.
-
 [source, text]
 ----
 config_ordinal = 120
 com.acme.myproject.someserver.url = http://more_important.server/some/endpoint
 ----
 
+Note that `config_ordinal` can be set within any `ConfigSource` implementation.  The default implementat of `getOrdinal()` will attempt to read this value, if found and a valid integer, the value will be used, otherwise the default of `100` will be returned.
```

Comment:
typo implementation. I would rephrase this to make it readable. In general, it is ok.